cmd/scollector/doc.go

```diff
@@ -24,6 +24,9 @@ The flags are:
 	-f=""
 		Filters collectors matching these terms, separated by
 		comma. Overrides Filter in conf file.
+	-F=""
```

Comment:
We have been moving a lot of things into the conf file only, and removing the flag way to do them. Do you have a compelling reason why this needs to be a flag, and not just specified in the config file? What do you think @gbrayut?